source/clear-linux/reference/bundles/bundles.rst

```diff
@@ -3,7 +3,9 @@
 Available bundles
 #################
 
-This document provides a current list of `available bundles`_. To learn more about how |CL-ATTR| uses bundles for software deployment, visit :ref:`bundles-about`.  
+This document provides a current list of `available bundles`_. To learn more about how |CL-ATTR| uses bundles for software deployment, visit :ref:`bundles-about`.
```

Comment:
The first sentence is describing the current page content, yet the link in the first sentence links off-site to a different list (plus the link text is the same text as the page title). I think this is misleading to the user (you click on something in the sentence describing the current page, but end up on another site).

Perhaps simply unlink the text in the first sentence, as the bundle/pundle names in the list below link to the respective repos (as you call out in your update :) )

Reply:
Done.